src/main/java/talker/Storage.java

```diff
@@ -0,0 +1,85 @@
+package talker;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import talker.task.Task;
+import talker.task.TaskList;
+
+/**
+ * Represents an object to read inputs and write outputs to file
+ */
+public class Storage {
+
+    // filepath for file to read/write into
+    private Path filePath;
+    //directory path for directory file read/write into is in
```

Comment:
```suggestion
    // directory path for directory file read/write into is in
```
just a small space between